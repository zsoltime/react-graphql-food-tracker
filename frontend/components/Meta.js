import Head from 'next/head';
import React from 'react';

const Meta = () => (
  <Head>
    <meta
      name="viewport"
      content="width=device-width, initial-scale=1"
    />
    <meta charSet="utf-8" />
    <link rel="shortcut icon" href="/static/favicon.png" />
    <title>Food Tracker</title>
  </Head>
);

export default Meta;
