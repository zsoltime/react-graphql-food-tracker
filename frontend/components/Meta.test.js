import React from 'react';
import { shallow } from 'enzyme';
import toJSON from 'enzyme-to-json';

import Meta from './Meta';

describe('<Meta />', () => {
  it('matches snapshot', () => {
    const wrapper = shallow(<Meta />);

    expect(toJSON(wrapper)).toMatchSnapshot();
  });
});
