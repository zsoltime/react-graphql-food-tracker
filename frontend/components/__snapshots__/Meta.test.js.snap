// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Meta /> matches snapshot 1`] = `
<SideEffect(Head)>
  <meta
    content="width=device-width, initial-scale=1"
    name="viewport"
  />
  <meta
    charSet="utf-8"
  />
  <link
    href="/static/favicon.png"
    rel="shortcut icon"
  />
  <title>
    Food Tracker
  </title>
</SideEffect(Head)>
`;
